src/PlaydateForm/PlaydateForm.js

```diff
@@ -23,13 +23,23 @@ class PlaydateForm extends Component {
   handleSubmit = (event) => {
     event.preventDefault()
     const { date, location } = this.state
-    const { currentUserId, selectedUserId, addPlaydateForSelectedUser } = this.props
+    const { currentUserId, selectedUserId, updateProfile, updateCurrentUser } = this.props
     const usersInvolved = [ currentUserId, selectedUserId ]
-    const newPlaydate = {
-      id: Date.now(),
-      date, location, usersInvolved
-    }
-    addPlaydateForSelectedUser(newPlaydate)
+    const playdateId = Date.now()
+    const playdates = usersInvolved.map(id => {
+      return {
+        id: playdateId,
```

Comment:
Why does each `playdate` get the same `id` with `const playdateId = Date.now()`? Do you need the `id: playdateId` to be unique in your logic?